src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+header {
```

Comment:
Avoid using tag names for styling. Instead, use a class or ID selector for the header element.